modules/fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/fs2/Environment.scala

```diff
@@ -159,6 +156,9 @@ object Environment {
   def makePause[F[_]: Concurrent]: Resource[F, SignallingRef[F, Boolean]] =
     Resource.make(SignallingRef(true))(_.set(true))
 
+  private def metricsReporter[F[_]: Sync: ContextShift: Timer](blocker: Blocker, config: ConfigFile): F[Metrics[F]] =
+    config.metrics.map(Metrics.build[F](blocker, _)).getOrElse(Metrics.noop[F].pure[F])
+
```

Comment:
```suggestion
  private def metricsReporter[F[_]: Sync: ContextShift: Timer](blocker: Blocker, config: ConfigFile): F[Metrics[F]] =    
    Metrics.build[F](blocker, config.metrics)
```

Just a proposal. That's how I do it for [RDB loader](https://github.com/snowplow/snowplow-rdb-loader/blob/feature/metrics/modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/dsl/StatsD.scala#L40-L58). I don't know what the best approach is, it might be just a matter of taste.